ide/libs.flexmark/nbproject/project.xml

```diff
@@ -26,8 +26,51 @@
             <code-name-base>org.netbeans.libs.flexmark</code-name-base>
             <module-dependencies/>
             <public-packages>
-                <subpackages>com.vladsch.flexmark</subpackages>
```

Comment:
Any reason behind of this change?

Generally I'm in favor of the exact specification of the public packages in modules, though library modules I'd just use the `subpackages`.

Reply:
Frankly, I don't know if this is really necessary, IDE generated it.
Can i remove this packages declarations and keep only subpackages?